database/migration/src/main/resources/changesets/embiggen_metadata_value.xml

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.3.xsd">
+
+    <changeSet author="mcovarr" id="entry_or_journal_existence_xor">
+        <preConditions>
```

Comment:
tl;dr it looks like it needs to stay the way it is.

It turns out that if the precondition is put at the changelog level, Liquibase will try to run it before running any changesets.  Since neither METADATA_JOURNAL nor METADATA_ENTRY would exist on a new database, the overall migration would incorrectly fail.